src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -641,15 +641,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Do
                     Dim hitNonoverloadingSymbol As Boolean = False
 
-                    Dim currentResult = LookupResult.GetInstance()
+                    Dim currentResult As LookupResult = Nothing
                     LookupWithoutInheritance(currentResult, currentType, name, arity, options, accessThroughType, binder, useSiteDiagnostics)
-                    If result.IsGoodOrAmbiguous AndAlso currentResult.IsGoodOrAmbiguous AndAlso Not LookupResult.CanOverload(result.Symbols(0), currentResult.Symbols(0)) Then
+                    If result IsNot Nothing AndAlso result.IsGoodOrAmbiguous AndAlso currentResult IsNot Nothing AndAlso currentResult.IsGoodOrAmbiguous AndAlso Not LookupResult.CanOverload(result.Symbols(0), currentResult.Symbols(0)) Then
```

Comment:
Nit: consider breaking this long line

Reply:
➡️ Resolved by removing the need for the additional null checks that increased the line length